recipes/dynare_preprocessor/meta.yaml

```
@@ -0,0 +1,41 @@
+package:
+  name: dynare-preprocessor
+  version: 0.0.0.dev0.2a63840eb534c3dcd1f50477d935ecc92294f00b
+
+source:
+  url: https://git.dynare.org/Dynare/preprocessor/-/archive/6235e6907526d468e2c35f14c38db1e308698bc9/preprocessor.tar.gz
+  sha256: 06f76245ff5c6ac0811438cea6b817a321dc702c188b680e2a3191b54837c4ab
+
+build:
+  number: 0
+  skip: true  # [win or osx]
```

```suggestion
{% set build = 1 %}
{% set commit = 6235e6907526d468e2c35f14c38db1e308698bc9 %}

package:
  name: dynare-preprocessor
  version: 0.0.0.dev{{ build }}.{{ commit }}

source:
  url: https://git.dynare.org/Dynare/preprocessor/-/archive/{{ commit }}/preprocessor.tar.gz
  sha256: 06f76245ff5c6ac0811438cea6b817a321dc702c188b680e2a3191b54837c4ab

build:
  number: {{ build }}
  skip: true  # [win or osx]
```
The build number and the version string need to by programatically linked.